### PR TITLE
Add some fixes

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

File: winfsp/src/filesystem/context.rs [hunk 1]
```diff
@@ -67,18 +67,18 @@ pub trait FileSystemContext: Sized {
     ///    Ok(security)
     /// }
     /// ```
-    fn get_security_by_name<P: AsRef<U16CStr>>(
+    fn get_security_by_name(
```
Comment (chyyran) [inline winfsp/src/filesystem/context.rs:70]: Please drop your changes to `FileSystemContext` in [190fa78](https://github.com/SnowflakePowered/winfsp-rs/pull/9/commits/190fa784b5f29aaa43c06fd9c89ddc6f9682904a). It is intentionally `AsRef<U16CStr>>` to allow for potentially lossless conversions between string type representations such like `HSTRING`

**Review by chyyran — CHANGES_REQUESTED** [review]

File: winfsp/src/filesystem/context.rs (hunk 1 above)
Comment (TimeEngineer) [reply winfsp/src/filesystem/context.rs:70]: When do you need to call these methods, except in the interface ? 🤔

**Comment by TimeEngineer** [comment]
It adds support for `aarch64-pc-windows-msvc` which was missing

**Comment by chyyran** [comment]
- Superceded by #15

**Pull request closed without merging**